lib/atlas/scaler.rb

```diff
@@ -54,5 +55,10 @@ def copy_etengine_data_files
           map { |subdir| File.join(@base_dataset.dataset_dir, subdir) },
         @derived_dataset.dataset_dir)
     end
+
+    def create_empty_graph_values_file
+      File.write(@derived_dataset.dataset_dir.
+        join(Dataset::Derived::GRAPH_VALUES_FILENAME), "--- {}")
```

Comment:
This convinces me that the graph values should be wrapped in a helper class. IMO `Scaler` shouldn't have to worry about what the graph values file contains (YAML, JSON, binary data) and probably not even *where* it is located.

```ruby
# This could be...
GraphValues.new(path).save
# or
GraphValues.create(path)
# or ... ?
```